mobile-app/app/api/transaction/dfi_converter.ts

```diff
@@ -38,21 +38,16 @@ export async function dfiConversionSigner (account: WhaleWalletAccount, amount:
 }
 
 export function dfiConversionCrafter (amount: BigNumber, mode: ConversionMode, onBroadcast: () => any, submitButtonLabel?: string): DfTxSigner {
+  const [symbolA, symbolB] = mode === 'utxosToAccount' ? ['UTXO', 'DFI'] : ['DFI', 'UTXO']
   return {
     sign: async (account: WhaleWalletAccount) => await dfiConversionSigner(account, amount, mode),
     title: translate('screens/ConvertConfirmScreen', 'Converting DFI'),
-    description: translate('screens/ConvertConfirmScreen', 'Converting {{amount}} {{symbolA}} to {{symbolB}}', {
-      amount: amount.toFixed(8),
-      ...(mode === 'utxosToAccount'
-        ? {
-          symbolA: 'UTXO',
-          symbolB: 'Token'
-        }
-        : {
-          symbolA: 'Token',
-          symbolB: 'UTXO'
-        })
-    }),
+    drawerMessages: {
+      preparing: translate('screens/ConvertConfirmScreen', 'Preparing to convert…'),
+      waiting: translate('screens/ConvertConfirmScreen', 'Converting DFI…'),
+      complete: translate('screens/ConvertConfirmScreen', 'Transaction completed')
+    },
+    description: translate('screens/ConvertConfirmScreen', 'Converting {{amount}} {{symbolA}} to {{symbolB}}', { amount: amount.toFixed(8), symbolA, symbolB }),
```

Comment:
want to double check do we need to update this description too? This will not appear in ocean interface but in the passcode prompt right?